proxyd/cmd/proxyd/main.go

```diff
@@ -73,9 +73,10 @@ func main() {
 	}
 
 	// Register after start.
-	externalListenAddr := config.Nacos.ExternalListenAddr
-	if os.Getenv(nacosExternalListenAddr) != "" {
-		externalListenAddr = os.Getenv(nacosExternalListenAddr)
+	externalIP := config.Nacos.ExternalIP
```

Comment:
Move this code under `len(config.Nacos.URLs) > 0` will be better.

Reply:
Done